### ISSUE TITLE
When testing the GameObjectFactory, there was a bug with itemIDs and buffer lengths not making sense. Needs to fix this ASAP.

### ISSUE THREAD

**Issue by KarlClinckspoor** [KarlClinckspoor/UnderworldRandomizer#41]

**Comment by KarlClinckspoor** [comment]
I was comparing Krokot's Ultimate Editor with Hank's UnderworldEditor and I found a few things:

* When I load krokot's lev.ark, without any modifications being made (just load, save, exit), and I opened this in hank's, every object in the object list in hank's becomes "a hand axe", meaning it's all zeroes. I'm guessing here, but is this related to the "Free list of static/mobile objects"? Those have a weird format. It's a counter and either 0, 1, 2 or 3... So krokot's editor modifier the buffer and apparently that doesn't impede the game from loading correctly.
* The data that is in the places where krokot's sets to 0 appears to be completely random at first glance. Like, the first 256 objects are supposed to store mobile info, but that's not the case. Like, at pos 1, itemID tells it's a jeweled shield, post2 it's a black sword. Those are static objects, not mobile. That's why I thought there was a bug in my code getting itemID -- there isn't necessarily a correlation between itemID and buffer length, given this set of "noise".
* Apparently, the correct way forward is to figure out *which* data to load as something meaningful, and which data to load but "ignore" the content. In the beginning, with simpler randomizers, this won't be an issue because I won't be adding/removing objects. However, when things get spicier, I'll need to deal with this issue.

I've made some tests that prove, at least the itemID, that my object loading functions are correct - or at least match hank's editor (good enough for me). First test at commit #4d58be9. However, there appears to be a problem somewhere and I'm not getting every id right of blocks after the first one.
